src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -435,9 +442,24 @@ Status GcsActorManager::RegisterActor(
   // The backend storage is supposed to be reliable, so the status must be ok.
   RAY_CHECK_OK(gcs_table_storage_->ActorTable().Put(
       actor->GetActorID(), *actor->GetMutableActorTableData(),
-      [this, actor](const Status &status) {
+      [this, actor, actor_already_destroyed_callback](const Status &status) {
         // The backend storage is supposed to be reliable, so the status must be ok.
         RAY_CHECK_OK(status);
+        // If an owner dies before this callback is called, the actor could have been
```

Comment:
The owner and the creator of this actor is the same core worker?

Reply:
Yes. We should probably use the same term. I will use the creator instead of owner! Will fix the comment.

Reply:
If the creator is dead, do we still need to send it a reply?

Reply:
Yeah that's a really great point. I will fix it :)